decidim-admin/spec/features/manage_participatory_process_steps_spec.rb

```diff
@@ -120,4 +122,30 @@
       end
     end
   end
+
+  context "activating a step" do
+    it "activates a step" do
+      within find("tr", text: process_step.title["en"]) do
```

Comment:
We should probably create a `translate` test helper that automatically does that. I can see the internals of translations changing in the future.